redisai/dag.go

```diff
@@ -8,6 +8,8 @@ type DagCommandInterface interface {
 	TensorSet(keyName, dt string, dims []int64, data interface{}) DagCommandInterface
 	TensorGet(name, format string) DagCommandInterface
 	ModelRun(name string, inputTensorNames, outputTensorNames []string) DagCommandInterface
+	ModelExecute(name string, inputs, outputs []string, timeout int64) DagCommandInterface
+	ScriptExecute(name, fn string, keys, inputs, inputArgs, outputs []string, timeout int64) DagCommandInterface
```

Comment:
Consider renaming to `(name, fn string, inputKeys, inputTensors, inputArgs, outputs []string, timeout int64)` or to 
`(name, fn string, keys, inputs, args, outputs []string, timeout int64)`